### ISSUE TITLE
Create a function for the visualization of the parts in test SDF

### ISSUE THREAD

**Issue by ern3509** [ern3509/Part-Guided-HyperDiffusion#9]
After the overfitting of the MLP, we should be able to visualize the part of the object separately on jointly. 
Hints: The output of the model provide the logits for each class. Find a way to get the part out of it.
Concatenate the coords, the occ and the label together and apply the function for visualization
Check the file part_to_ply which can help here.
